### PR TITLE
Exec summary fix

### PR DESCRIPTION
**Pull request by skyanth**
Hi @PeterMosmans,

Can you please vet the changes in docbuilder.py in this commit? With these changes, the -execsummary option for docbuilder works. This completely deprecates the summary.xslt by the way...

### CODE REVIEW

File: chatops/python/docbuilder.py [hunk 1]
```diff
@@ -234,7 +234,7 @@ def main():
             except OSError as exception:
                 print_exit('[-] ERR: {0}'.format(exception.strerror),
                            exception.errno)
-        if options['execsummary'] == 'true':  # we're generating a summary as well as a report
+        if options['execsummary'] == True:  # we're generating a summary as well as a report
```
Comment (PeterMosmans) [inline chatops/python/docbuilder.py]: In the future, we can refactor this, as execsummary is an 'add-on': first build the execsummary, and don't use the `else` statement for building the report itself.

Comment (PeterMosmans) [inline chatops/python/docbuilder.py]: (lazy Python programmers leave out the comparison to True :wink: - true pythonista style:
`if options['execsummary`]:` as it's already a boolean test)

**Comment by PeterMosmans** [comment]
superseded by pull request #40

File: chatops/python/docbuilder.py (hunk 1 above)
Comment (skyanth) [inline chatops/python/docbuilder.py]: I know! *Every time* I see this in other people's code I think: man, I have to do that too. And then I don't. I'm such an imposter. ;)

**Pull request merged**